docs/site-example/tests/validate_site.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+set -uo pipefail
+
+# validate_site.sh checks if the site.conf is valid json
+GLUON_REPO="https://github.com/rubo77/gluon"
+GLUON_BRANCH='ssid-changer'
```

Comment:
While this is just an example, a config-file or passing these variables via CLI might be the better solution.